src/Components/List/index.jsx

```diff
@@ -23,13 +27,44 @@ function List ({ list, toggleComplete }) {
     return (
         <>
         {displayList.map(item => (
-          <div key={item.id}>
-            <p>{item.text}</p>
-            <p><small>Assigned to: {item.assignee}</small></p>
-            <p><small>Difficulty: {item.difficulty}</small></p>
-            <div onClick={() => toggleComplete(item.id)}>Complete: {item.complete.toString()}</div>
-            <hr />
-          </div>  
+          <Card mb="sm" shadow="md" withBorder key={item.id}>
+            <Card.Section withBorder>
+              <Group position="apart">
+                <Group>
+                  <If condition={isLoggedIn && can('update')}>
+                    <Then>
+                      <Badge
+                      onClick={() => toggleComplete(item.id)}
+                      color={item.complete ? 'red' : 'green'}
+                      variant="filled"
+                      m="3px"
+                      >
+                        {item.complete ? 'Complete' : 'Pending'}
+                      </Badge>
+                    </Then>
+                    <Else>
+                      <Badge
+                      color={item.complete ? 'red' : 'green'}
+                      variant="filled"
+                      m="3px"
+                      >
+                        {item.complete ? 'Complete' : 'Pending'}
+                      </Badge>
+                    </Else>
+                  </If>
```

Comment:
This is a fair amount of repeated code for one small change; in fact, they will look the exact same. To avoid repeating yourself, I would move the condition to the `toggleComplete` function, or even to the callback here.
```js
onClick={can('update') ? () => toggleComplete(item.id) : null}